tests/workers/xmodel/test_resnet50.py

```diff
@@ -17,6 +17,7 @@
 import amdinfer
 import amdinfer.pre_post as pre_post
 import amdinfer.testing
+from amdinfer._amdinfer import *
```

Comment:
Don't import * and you shouldn't need _amdinfer either

Reply:
resolved